client/reader/following-manage/index.jsx

```diff
@@ -137,6 +137,24 @@ class FollowingManage extends Component {
 		);
 	};
 
+	reportFollowByUrlRender = siteUrl => {
+		if ( siteUrl ) {
+			recordTrack( 'calypso_reader_following_manage_follow_by_url_render', {
+				url: siteUrl,
+			} );
+		}
+	};
+
+	shouldShowFollowByUrl = () => resemblesUrl( this.props.sitesQuery );
+	componentDidUpdate() {
+		if (
+			this.shouldShowFollowByUrl
```

Comment:
yea yea, the whole thing was blocked by mergin https://github.com/Automattic/wp-calypso/pull/14379